src/JBrowse/QueryParamConfigMapper.js

```diff
@@ -1,8 +1,16 @@
 define(['dojo/_base/declare', 'JBrowse/Util/dot-object'], function (declare, dotObject) {
     return declare(null, {
 
-        constructor: function () {
+        constructor: function () { },
 
+        removeNullsFromArray: function(inputArray){
+                for (var i = 0; i < inputArray.length; i++) {
+                    if (inputArray[i] == null) {
+                        inputArray.splice(i, 1);
+                        i--;
+                    }
+                }
+                return inputArray;
```

Comment:
what about something like `array.filter( inputArray, function(el) { return el !== null })`.  from `dojo/_base/array`

Reply:
sure, if you want to be smart about it.